src/Native/LdaNative/model_block.cpp

```diff
@@ -157,7 +170,7 @@ namespace lda
         if (fullSparse)
         {
             // use a very large threshold to ensure every row of word-topic-table using a sparse representation
-            hot_thresh = std::numeric_limits<int>::max();
+            hot_thresh = numeric_limits<int>::max();
```

Comment:
>int [](start = 40, length = 3)

why not max of int32_t?

Reply:
indeed, why not?  - fixed.

---
In reply to: [276830864](https://github.com/dotnet/machinelearning/pull/3370#discussion_r276830864) [](ancestors = 276830864)